tests/nonbonded/test_nonbonded_interaction_group.py

```diff
@@ -116,7 +116,7 @@ def ref_ixngroups(conf, params, box, lamb):
         conf,
         params,
         example_box,
-        lamb=lamb,
+        [lamb],
```

Comment:
I think we could get a speedup here too by passing multiple lambdas here instead of using the `parametrize` decorator. (I had already done this for the nonbonded tests that weren't using `compare_forces`)

Reply:
Resolved in [775576f](https://github.com/proteneer/timemachine/pull/699/commits/775576ffea4347658d94d4599ccd8d22817b36d5)